src/wavm/mpi.cpp

```diff
@@ -647,14 +666,16 @@ WAVM_DEFINE_INTRINSIC_FUNCTION(env,
 
 WAVM_DEFINE_INTRINSIC_FUNCTION(env, "MPI_Abort", I32, MPI_Abort, I32 a, I32 b)
 {
-    faabric::util::getLogger()->debug("S - MPI_Abort {} {}", a, b);
+    ContextWrapper ctx;
+    ctx.getMpiLogger()->debug("S - MPI_Abort {} {}", a, b);
     //        return terminateMpi();
```

Comment:
Not part of this change but we should probably think about what terminating MPI means, whether we can clean up anything.